docs/xss.rst

```diff
@@ -0,0 +1,95 @@
+Performing XSS Vulnerability Audits
+==================================
+
+The bok-choy framework includes the ability to perform XSS (cross-site scripting) audits on
+web pages using a short XSS locator defined in
+https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet#XSS_Locator.
+
+Leverage Your Existing Tests and Fail on XSS Vulnerability Errors
+-------------------------------------------------------------
+
+To trigger XSS vulnerability audits passively, set the ``VERIFY_XSS``
+environment variable to ``True``. Doing so triggers an XSS audit
+whenever a page object's ``q`` method is called. If errors are
+found on the page, an XSSExposureError is raised.
+
+.. note:: XSS vulnerabilities will only be found if you have injected the specific string
+XSS_INJECTION defined in bok_choy.page_object into your page content.
+
+You might already have some bok-choy tests written for your web application.
+Here is an example of a bok-choy test that will implicitly check for XSS vulnerabilities.
+It involves clicking on a button on page, and it is assumed that the user's name is
```

Comment:
"It involves clicking a button to display the user's name on the page." (Is this still accurate?)